pkgs/by-name/ed/edcas-client/package.nix

```
@@ -0,0 +1,82 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, cmake
+, wayland
+, libX11, libXcursor, libXi, libXrandr
+, libxkbcommon, libxcb
+, libglvnd
+}:
+
+let
+  rpathLibs = [
+    libglvnd
+    libXcursor
+    libXi
+    libxkbcommon
+    libXrandr
+    libX11
+    wayland
+  ];
+in rustPlatform.buildRustPackage rec {
+  pname = "edcas-client";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "arne-fuchs";
+    repo = pname;
```

`repo` should not depend on `pname`
```suggestion
    repo = "edcas-client";
```